libs/portal-integration-angular/src/lib/core/components/search-criteria/search-criteria.component.ts

```diff
@@ -40,18 +40,18 @@ export class SearchCriteriaComponent extends PageHeaderComponent implements OnIn
   }
 
   /** Event emitted when the search button has been pressed. */
-  @Output() readonly search: EventEmitter<'basic' | 'advanced'> = new EventEmitter<'basic' | 'advanced'>()
+  @Output() readonly search: EventEmitter<'simple' | 'advanced'> = new EventEmitter<'simple' | 'advanced'>()
```

Comment:
Please do not change the names in the code because it will break existing applications using it